### ISSUE TITLE
[enhancement] invert Dual page split

### ISSUE THREAD

**Issue by Riztard** [tachiyomiorg/tachiyomi#4595]
## Device information
* Tachiyomi version: Prev r3130
* Android version: 10
* Device: Redmi 8

## Steps to reproduce
1. read manga A
2. turn on invert dual page split placement
3. read manga B

### Expected behavior
on manga B "invert" should be off 

### Actual behavior
on manga B invert dual page is on 

## Other details
if changed while reading should not change for all/global
only if changed in the setting it should behave as global setting

**Comment by scb261** [comment]
It's more feature request than bug. And looks like duplicate of #452

**Comment by ghostbear** [comment]
Yes, this is a feature request, because it's a global setting not a per series setting currently. I would say it's currently a pain in the butt to implement per series settings. I think if #3522 where to get merged it would allow implementing per series settings more easier
